src/main/resources/reference.conf

```diff
@@ -451,14 +451,17 @@ resourceTypes = {
       status = {
         description = "view details and configuration of the kubernetes application"
       }
+      stop_start = {
```

Comment:
why not 2 actions: start and stop?

Reply:
I didn't think there was a situation where one should be able to stop and not start an app (or vice versa). Runtimes work similarly: https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L399

(We grant the "manager" role the ability to delete, but not stop/start other users' apps.)

Reply:
I could make it 2 actions if that's preferred though

Reply:
I made it 2 actions, probably makes more sense

Reply:
never know what the future holds :)